src/gluonts/transform/feature.py

```diff
@@ -99,6 +99,8 @@ class LastValueImputation(MissingValueImputation):
     """
 
     def __call__(self, values: np.ndarray) -> np.ndarray:
+        if len(values) == 1:
+            return DummyValueImputation()(values)
```

Comment:
This would need to be converted to a float I guess

```suggestion
            return DummyValueImputation()(values.item())
```

Reply:
No it should be numpy array: `def __call__(self, values: np.ndarray) -> np.ndarray:`. 

A simple example (copying what the dummy value imputation does) with numpy array:
```
values = np.array([1])
nan_indices = np.where(np.isnan(values))
values[nan_indices] = 0
print(values)
[1]
```

and with float:
```
values = np.array([1]).item()
nan_indices = np.where(np.isnan(values))
values[nan_indices] = 0
print(values)

---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-97-8448c7edc37d> in <module>
      1 values = np.array([1]).item()
      2 nan_indices = np.where(np.isnan(values))
----> 3 values[nan_indices] = 0
      4 print(values)

TypeError: 'int' object does not support item assignment

```

Reply:
yes, sorry, I got confused between __init__ and __call__